wsmaster/che-core-api-devfile/pom.xml

```diff
@@ -191,4 +191,133 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+    <profile>
+        <id>devfile-docs-docker</id>
```

Comment:
Maybe it would be better to move Devfile docs generation to a separate maven module. WDYT?

Reply:
It anyway will have to contain that profiles (to allow different build types). And they are incactive by default. So we will have module which will do nothing 99% of time.